### PR TITLE
[CS-3706]: Fix find undefined error and reorder sentry logs

### PR DESCRIPTION
**Pull request by danibonilha**
### Description
Giving where this error was happening, it seems it's been happening for a while, but with the sentry update, sentry captures the exception better and now we are able to see, this fixes the issue and also reorders the logs, to log the breadcrumb before capturing some exceptions.

### CODE REVIEW

File: cardstack/src/redux/collectibles.ts [hunk 1]
```diff
@@ -172,6 +172,7 @@ const fetchNFTsViaOpenSea = () => (
     } catch (error) {
       dispatch({ type: COLLECTIBLES_FETCH_FAILURE });
       captureException(error);
+      logger.sentry('COLLECTIBLES_FETCH_FAILURE');
```
Comment (oidoug) [inline cardstack/src/redux/collectibles.ts:175]: Doesn't it make sense to have the capture before the log here too?

File: src/redux/fallbackExplorer.js [hunk 2]
```diff
@@ -83,7 +83,7 @@ const fetchCoingeckoIds = async (network, coingeckoCoins) => {
     const honeyswapTokens = data.tokens;
 
     honeyswapTokens.forEach(({ address: tokenAddress, symbol }) => {
-      const coingeckoToken = coingeckoCoins.find(
+      const coingeckoToken = coingeckoCoins?.find(
```
Comment (oidoug) [inline src/redux/fallbackExplorer.js:86]: 👏

**Review by oidoug — APPROVED** [review]
LGTM, just a minor question.

File: cardstack/src/redux/collectibles.ts (hunk 1 above)
Comment (danibonilha) [reply cardstack/src/redux/collectibles.ts:175]: it does! 😅, good catch.

**Pull request merged**